machine.go

```diff
@@ -458,6 +459,14 @@ func (m *Machine) SetScratch(scratchsize int64, path string) {
 	}
 }
 
+func (m *Machine) SetInitrdPath(path string) {
```

Comment:
I don't see the benefit of changing the initrd path since it's generated on demand by fakemachine?

Reply:
I agree, let remove it